python/Ganga/new_tests/GPI/GangaList/TestMutableMethods.py

```diff
@@ -1,19 +1,14 @@
-from GangaTest.Framework.tests import GangaGPITestCase
-from Ganga.GPIDev.Base.Proxy import isProxy, isType, TypeMismatchError
-from Ganga.GPIDev.Base.Proxy import ReadOnlyObjectError
-from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList as gangaList
-from Ganga.GPIDev.Lib.GangaList.GangaList import decorateListEntries
-from Ganga.GPIDev.Adapters.IGangaFile import IGangaFile
-
-from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList
-
 import random
 import string
 import time
 
+from ..GangaUnitTest import GangaUnitTest
+from Ganga.GPIDev.Base.Proxy import isProxy, isType, TypeMismatchError
```

Comment:
I'd prefer to see these imports made within the test class as py.test can't know where to find these on import and checking the class until the path has been manipulated by the GangaUnitTest constructor.

That or we should fix the GangaUnitTest to modify the sys.path on import which we're currently not doing.